Dockerfile

```diff
@@ -2,6 +2,7 @@
 FROM docker.io/golang:1.17 as builder
 ARG GIT_VERSION="(unset)"
 ARG COMMIT_ID="(unset)"
+ARG ARCH="(unset)"
```

Comment:
The string "(unset)" has no special meaning and I used it to pass strings to the go binary so we'd know if the build never specified the GIT version and id. However I'm not sure it makes sense to set ARCH to "(unset)"  would it make more sense to default to empty string "" or "amd64" ?  Just so it defaults to something valid?

Reply:
I was wondering about it as well. I would vote for empty string. Will fix.